src/store.ts

```diff
@@ -0,0 +1,18 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers/rootReducer';
+
+declare global {
+    interface Window {
+        __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose;
```

Comment:
This is to set up on TS the redux DevTools. For JS it will be easier, but TS doesnt know the type of that property, and we will get the following error:

Property ‘__REDUX_DEVTOOLS_EXTENSION_COMPOSE__’ does not exist on type ‘Window’.

Hence, we need to add the property to the global window as above to make it work.